common/artifacts/901_wc_artifacts_weapon_onehanded.txt

```diff
@@ -78,6 +79,31 @@ ironfoe = {
 
 	slot = weapon
 }
+the_kingslayer = {
+	quality = 4
+	flags = { combat_weapon sword }
+	
+	indestructible = no
+	stacking = no
+	
+	active = {
+		shared_physical_weapon_trigger = yes
+		is_adult = yes
+	}
+	
+	martial = 1
+	intrigue = 1
+	combat_rating = 17
+	
+	human_group_opinion -7
+	creature_orc_opinion = 7
```

Comment:
So even if a human owns the weapon, he'll be hated by humans and loved by orcs for it? Why?